modal/mount.py

```diff
@@ -670,19 +691,19 @@ def _get_client_mount():
     )
 }
 
+
 SYS_PREFIXES |= {p.resolve() for p in SYS_PREFIXES}
 
+MODAL_PACKAGES = ["modal", "modal_proto", "modal_version"]
+
 
 def _is_modal_path(remote_path: PurePosixPath):
     path_prefix = remote_path.parts[:3]
     remote_python_paths = [("/", "root"), ("/", "pkg")]
     for base in remote_python_paths:
-        is_modal_path = path_prefix in [
-            base + ("modal",),
-            base + ("modal_proto",),
-            base + ("modal_version",),
-            base + ("synchronicity",),
-        ]
+        is_modal_path = path_prefix in [base + (mod,) for mod in MODAL_PACKAGES] or path_prefix == base + (
+            "synchronicity",
+        )
```

Comment:
Poor synchronicity — not considered part of the family, but doesn't get its own pip downloads.

Reply:
😢